pkg/minikube/node/cache.go

```diff
@@ -107,7 +108,15 @@ func beginDownloadKicArtifacts(g *errgroup.Group, driver string, cRuntime string
 			out.T(out.Pulling, "Pulling base image ...")
 			g.Go(func() error {
 				glog.Infof("Downloading %s to local daemon", baseImage)
-				return image.WriteImageToDaemon(baseImage)
+				err := image.WriteImageToDaemon(baseImage)
+				if err != nil {
+					// TODO : remove this global set when this issue is closed
+					// https://github.com/kubernetes/minikube/issues/7944
+					viper.Set("base-image", kic.BaseImageFallBack)
```

Comment:
This feels really bad - I would rather see the global fixed in a PR before merging this one.

Reply:
the issue that was set to fix that was evaluated as Backlog priority. so that would be a longer wait to get this PR merged.

Reply:
@medyagh - that's OK. I would rather see this done correctly than quickly. Feel free to update priorities as you see fit.

Reply:
sounds good, here is a PR for that https://github.com/kubernetes/minikube/pull/7985 whenever you feel ready to review.

Reply:
done